backend/inspirehep/search/facets.py

```diff
@@ -177,6 +178,41 @@ def inner(values):
     return inner
 
 
+def curation_filter(author_recid):
+    filter_for_option = {
+        "Claimed papers": {
+            "nested": {
+                "path": "authors",
+                "query": {
+                    "bool": {
+                        "must": [
+                            {"term": {"authors.curated_relation": True}},
+                            {"term": {"authors.record.$ref": author_recid}},
+                        ]
+                    }
+                },
+            }
+        },
+        "Unclaimed papers": {
+            "nested": {
+                "path": "authors",
+                "query": {
+                    "bool": {
+                        "must_not": [{"term": {"authors.curated_relation": True}}],
+                        "must": [{"term": {"authors.record.$ref": author_recid}}],
+                    }
+                },
+            }
+        },
+    }
+
+    def inner(values):
+        filters = [filter_for_option[value] for value in values]
+        return Q("bool", must=filters)
```

Comment:
```suggestion
        return Q("bool", filter=filters)
```
as filters are more performant but don't contribute to the score, see https://www.elastic.co/guide/en/elasticsearch/reference/7.1/query-filter-context.html for a more detailed explanation.